components/datadog/apps/logger/images/logger/README.md

```diff
@@ -0,0 +1,41 @@
+#  Logger app
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Thank you for adding a doc file!